api_examples/sequential_learning_api_tutorial.ipynb

```diff
@@ -0,0 +1,692 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# PyCC Sequential Learning Tutorial\n",
+    "\n",
+    "*Authors: Edward Kim, Enze Chen*\n",
+    "\n",
+    "In this notebook, we will cover how to perform **sequential learning** (SL) using the [Citrination API](http://citrineinformatics.github.io/python-citrination-client/). Sequential learning is the key workflow which allows machine learning algorithms and in-lab experiments to iteratively inform each other.\n",
+    "\n",
+    "To replace the need for an actual laboratory, this notebook uses a simple *toy function* that allows for \"measurements\" on the data.\n",
+    "\n",
+    "## Learning objectives\n",
+    "\n",
+    "By the end of this tutorial, you will know how to:\n",
+    "\n",
+    "- Use PyCC end to end, from uploading an initial dataset to running SL\n",
+    "- Understand how to configure SL parameters to match your laboratory setup\n",
+    "- Asynchronously issue SL jobs to Citrination's backend\n",
+    "\n",
+    "## Background knowledge\n",
+    "\n",
+    "In order to get the most out of this tutorial, you should already be familiar with the following:\n",
+    "\n",
+    "- Create and access datasets through the API ([documentation](http://citrineinformatics.github.io/python-citrination-client/tutorial/data_examples.html) and [tutorial](https://github.com/CitrineInformatics/community-tools/blob/master/api_examples/data_client_api_tutorial.ipynb))\n",
+    "- Create and access data views through the API ([tutorial](https://github.com/CitrineInformatics/community-tools/blob/master/api_examples/data_views_api_tutorial.ipynb))\n",
+    "- What the data views [front-end UI](https://citrination.com/data_views) looks like"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## BEFORE YOU RUN THIS NOTEBOOK\n",
+    "\n",
+    "This notebook uses some convenience functions to wrap several API endpoints. These are listed at the end of the notebook in the section titled **Convenience functions**. Run those functions first before running the main notebook!"
```

Comment:
Nice, I like how this is at the end. Might be nice to create an internal link to Convenience function header with syntax: `[Text we read](#Header-name-with-spaces-hyphenated)` and then a link at the end of Convenience function taking you back up to Imports.